tutorial/inputoutput.po

```diff
@@ -477,13 +476,13 @@ msgid ""
 "binary mode) are read and returned. If the end of the file has been reached, "
 "``f.read()`` will return an empty string (``''``).  ::"
 msgstr ""
-"Pour lire le contenu d'un fichier, appelez ``f.read(taille)`` : elle lit une "
+"Pour lire le contenu d'un fichier, appelez ``f.read(size)``, qui lit une "
 "certaine quantité de données et les donne sous la forme d'une chaîne (en "
-"mode texte) ou dans un objet *bytes* (en mode binaire). *taille* est un "
-"argument numérique optionnel. Quand *taille* est omis ou négatif, le contenu "
-"entier du fichier est lu et donné, c'est votre problème si le fichier est "
+"mode texte) ou dans un objet *bytes* (en mode binaire). *size* est un "
+"argument numérique optionnel. Quand *size* est omis ou négatif, le contenu "
+"entier du fichier est lu et renvoyé ; c'est votre problème si le fichier est "
 "deux fois plus gros que la mémoire de votre machine. Sinon, un maximum de "
-"*taille* octets sont lus et donnés. Lorsque la fin du fichier est atteinte, "
+"*size* octets sont lus et renvoyé. Lorsque la fin du fichier est atteinte, "
```

Comment:
Il manque la partie `*size* characters (in text mode) or` 😄

Reply:
C'set d'ailleurs l'objet du *fuzzy*.
Pour trouver les modifications qui ont amené le *fuzzy*, tu peux faire ceci : 
``git log tutorial/inputouput.po | head -n 30`` où tu vois qu'il y a une propagation (*commit* en anglais) de Julien Palard juste avant le merge 3.8. 
Notons la référence de cette propagation : ``517157af…``
Ensuite ``git difftool 517157af tutorial/inputoutput.po`` affiche les différences entre la version actuelle et celle avant le merge 3.8. Tu peux chercher la chaîne `fuzzy` dans la colonne de droite et les modifications dans les paragraphes apparaissent clairement.
Tu sais maintenant sur quoi concentrer tes efforts de traduction.

Reply:
Le *renvoyé* doit être mis au pluriel.